src/components/CountryTable.tsx

```diff
@@ -20,31 +22,77 @@ const CountryTable: React.FC<CountryTableProps> = ({ data }) => {
       </IonRow>
       <HeadRow />
       {data &&
-        Object.keys(data).map((key) => <ItemRow key={key} item={data[key]} />)}
+        Object.keys(data).map((key) => (
+          <ItemRow
+            style={
+              data[key] && data[key].CountryCode === myCoutryCode
+                ? "country-row"
+                : "item-row"
+            }
+            key={key}
+            item={data[key]}
+            defaultCountry={myCoutryCode}
```

Comment:
@tranthanhben2002 The `<ItemRow />` should not know about the logic of Your Country. You should determine it here and pass only props relevant to styling, not the business logic.
My edit proposal is:

```ts
Object.keys(data).map((key) => {
  const isUserCountry = data[key] && data[key].CountryCode === myCountryCode;
  return (
    <ItemRow
      className={isUserCountry ? "country-row" : "item-row"} 
      ...
      pinToTop={isUserCountry}
    />
  );
});
```

`pinToTop` means the row will be on top of the table, has border and bigger font size, etc. More about styling than business logic

Reply:
@tranthanhben2002 In addition, it's not ideal to display data using the HashMap `byCountries` directly here because we will need sorting feature. Should convert it to an array using `@computed` property (my other comment talked about this). Default the country with largest `Total Confirmed` should be on top (still below Your Country)